bin/console

```diff
@@ -42,7 +42,7 @@ $backend->initialize('Console');
 
 $application = new Application($kernel);
 $application->add(new \Console\Locale\ImportLocaleCommand());
-$application->add(new \Console\Thumbnails\GenerateThumbnailsCommand());
+$application->add(new \Console\Thumbnails\GenerateThumbnailsCommand($kernel->getContainer()->get('forkcms.utility.thumbnails')));
```

Comment:
maybe you can register it in the yml config instead of hardcoded here?

Reply:
Done